<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetTree</class>
 <widget class="QMainWindow" name="WidgetTree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WidgetTree</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>12</y>
      <width>831</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QLineEdit" name="leRootDir">
       <property name="text">
        <string>C:\dev\poco</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="2" column="0">
      <widget class="QTreeView" name="treeView">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pbDrawTree">
       <property name="text">
        <string>Построить дерево</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
